core/commands/pin/remotepin.go

```diff
@@ -669,3 +673,16 @@ func getRemotePinServiceInfo(env cmds.Environment, name string) (url, key string
 	}
 	return service.Api.Endpoint, service.Api.Key, nil
 }
+
+func normalizeEndpoint(endpoint string) (string, error) {
+	uri, err := neturl.ParseRequestURI(endpoint)
+	if err != nil || !strings.HasPrefix(uri.Scheme, "http") {
```

Comment:
Should we also error if there are query parameters or URL fragments?

Reply:
Added in 2ffdce500d5a788a19fbff1e1d4c8c5220bf3df2